src/app/components/EmptyState/EmptyState.stories.tsx

```diff
@@ -0,0 +1,26 @@
+import React from "react";
+
+import { ComponentStory, ComponentMeta } from "@storybook/react";
+import { EmptyState } from "@app/components/EmptyState/EmptyState";
+
+export default {
+  title: "Shared/EmptyState",
+  component: EmptyState,
+} as ComponentMeta<typeof EmptyState>;
+
+const Template: ComponentStory<typeof EmptyState> = (args) => (
+  <EmptyState {...args} />
+);
+
+export const EmptyStateBase = Template.bind({});
+EmptyStateBase.args = {
+  title: "No Smart Event instances",
+  createButton: {
+    title: "Create Smart Events instance",
+    onCreate: (): void => {},
+  },
+  quickStartGuide: {
+    i18nKey: "common.quickStartAccess",
+    onQuickstartGuide: (): void => {},
+  },
```

Comment:
For some reason the `i18nKey` is not converted properly to its corresponding string within the `EmptyState` story. I couldn't find why.
I guess It's not a big deal for now. Suggestions are obviously welcome.